### PR TITLE
JavaScript Capstone

### PR DESCRIPTION
**Pull request by malikhaiderkhan**
# 🎖️ ([JavaScript group capstone](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md))
## Highlights
✔️ Used fakestore API to create the website 
✔️ Used Involvement API for storing user Interactions 
✔️ Created a Home Page for showing different meals 
✔️ Retrieving fakestore data from API 
✔️ Used Involvement API on the home page to show user's likes count 
✔️ User can like a particular item 
✔️ When the User clicks on the comment button comment popup appears  
✔️ When the comment popup loads it shows item details and comments from different user 
✔️ User can also add new comments to the comments list  
✔️ Comment list also shows comments counter which is calculated using the DOM element counter 
✔️ bar under header shows the number of items in each category
 
## 👨‍💻 Technical Highlights
✔️ Used ES6 syntax 
✔️ Used fetch for retrieving data 
✔️ Consumed fakestore API 
✔️ Used Webpack 
✔️ Used jest for testing 
✔️ Unit Tested counters for various elements 
✔️ Used Linters for code quality assurance
 
## 📖 Professional Highlights
✔️ Used Kanban board for project tasks tracking 
✔️ Did the code review for peers 
✔️ Used git-flow Methodology

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <button class='btn btn-comment' data-url='https://fakestoreapi.com/products/${list.id}'>Comments <img src='${commentbtn}' /></button>
+            <button class='btn'>Check Item</button>
+        `;
+
+    productContainer.appendChild(productWrap);
+
+    const heartIcon = productWrap.querySelector('.likes');
+
+    // Fetch the number of likes from the API
+    fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/myodB0I2hWMN7rQnMtyn/likes/')
```
Comment (fobadara) [inline src/modules/display.js:34]: [[REQUIRED] It is a requirement to use async and await in your data fetching. Kindly refactor your code to use them.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good first attempt
✅ Use of try-catch blocks

## Required Changes ♻️
_Check the comments under the review._
I had to include some comments here because I could not access the file needed to make an inline comment

[REQUIRED] I searched your code base tirelessly but I could not find the code that counts the total movies on a page. Kindly implement this and put it in its own independent module. Please make sure that your tests also reflect this change.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sulaimonaa** [comment]
The count method is done on the categories,  the shopping products are in different categories and their total count is implemented on the category.js

**Comment by malikhaiderkhan** [comment]
Implemented all the required changes suggested by reviewer:

- implement the code that counts the total items/products on a page in category.js file.
- fix the license file.
- use await and async in data fetching as suggested by reviewer in display.js file.

File: README.md [hunk 2]
```diff
@@ -1 +1,154 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my sincere gratitude to Microverse for providing me with this opportunity to learn and grow as a developer. Their support and guidance have been invaluable throughout my journey.
+
+<br>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
[the remark this answers is not in the document]
Comment (rloterh) [reply README.md]: well resolved! ✔️

[the remark this answers is not in the document]
Comment (rloterh) [reply README.md]: well resolved! ✔️

File: src/modules/display.js (hunk 1 above)
Comment (rloterh) [reply src/modules/display.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Haider & Ayodeji,  

In my opinion, your project is ready for presentation! Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Counters properly implemented
🎲 Counters are tested!

Cheers..Keep rocking! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**